src/kw/kw.cpp

```diff
@@ -26,11 +26,14 @@ using std::string;
 
 
 
-KanoWorld::KanoWorld(std::shared_ptr<IHTTPClient> client) :
+KanoWorld::KanoWorld(const std::string& url,
+                     std::shared_ptr<IHTTPClient> client) :
     http_client(client),
     data_filename(string(getenv("HOME")) + "/" + ".mercury_kw.json"),
```

Comment:
Just something to thing about: mocking this `getenv` when it's used like this might be a bit awkward.